api/src/main/resources/log4j2-spring.xml

```diff
@@ -18,12 +18,13 @@
 <Configuration monitorInterval="60">
     <Properties>
         <Property name="log-path">logs</Property>
+        <Property name="cp-api-logger-hostname">$${env:MY_POD_NAME:-cp-api-srv}</Property>
```

Comment:
Shouldn't we change env var name here as well?

Reply:
Done